### PR TITLE
Support new layout format

### PR DESCRIPTION
**Pull request by gal-berger**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/27554

## Description
This PR handles:
1. allows importing the new layout version (layoutscontainer).
2. fix importing issue using 5.5

## Must have
- [ ] Unit Test or Example Code
- [ ] Changelog entry

### CODE REVIEW

File: tests/mocks_test.py [hunk 1]
```diff
@@ -226,3 +232,44 @@ def run():
 
     run()
     assert_reset()
+
+
+def test_import_layout(mocker):
```
Comment (yuvalbenshalom) [inline tests/mocks_test.py:237]: please add docs to all of the tests lice we do in content and sdk Its

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
looks good. see my comments.

File: demisto_client/demisto_api/api/default_api.py [hunk 2]
```diff
@@ -4288,11 +4279,11 @@ def import_layout_with_http_info(self, file, type, kind, **kwargs):  # noqa: E50
 
         form_params = []
         local_var_files = {}
-        if 'file' in params:
+        if params.get('file', None):
```
[the remark this answers is not in the document]
Comment (gal-berger) [reply demisto_client/demisto_api/api/default_api.py:4282]: yes.

File: tests/mocks_test.py (hunk 1 above)
Comment (gal-berger) [reply tests/mocks_test.py:237]: done

File: demisto_client/__init__.py [hunk 3]
```diff
@@ -244,3 +245,18 @@ def generic_request_func(self, path, method, body=None, **kwargs):
         _preload_content=params.get('_preload_content', True),
         _request_timeout=params.get('_request_timeout'),
         collection_formats=collection_formats)
+
+
+def get_url_for_demisto_version(api_client, params):
```
[the remark this answers is not in the document]
Comment (gal-berger) [reply demisto_client/__init__.py]: done

**Review by yuvalbenshalom — APPROVED** [review]

File: gen-code.sh [hunk 4]
```diff
@@ -44,6 +44,33 @@ import demisto_client/g' demisto_client/demisto_api/api/default_api.py
...
 sed -i '' -e 's#if six\.PY3:#if six.PY3 and r.getheader("Content-Type") != "application/octet-stream":#' demisto_client/demisto_api/rest.py
+
+# Fix import layout command
+start=`grep "verify the required parameter 'type'" demisto_client/demisto_api/api/default_api.py -n | cut -f1 -d: | tail -1 | tr -d "\\n"`
+end=`grep ".kind. when calling .import_layout." demisto_client/demisto_api/api/default_api.py -n | cut -f1 -d: | tail -1 | tr -d "\\n"`
+sed -i '' -e "$start,${end}d" demisto_client/demisto_api/api/default_api.py
+kind_line=`grep "'kind' in params:" -n demisto_client/demisto_api/api/default_api.py | cut -f1 -d: | tr -d "\\n"`
+start=$((kind_line-4))
+end=$((kind_line+1))
+sed -i '' -e "$start,$end s/if \(.*\) in params/if params.get(\1, None)/" demisto_client/demisto_api/api/default_api.py
```
Comment (amshamah419) [inline gen-code.sh:55]: This will change all lines in `default_api.py`. I'm okay with it, but is this intended?

Comment (amshamah419) [reply gen-code.sh:55]: Actually just caught the start and end boundary.

**Review by amshamah419 — APPROVED** [review]

**Comment by amshamah419** [comment]
@gal-berger @yuvalbenshalom If this is a showstopper for content, let's prepare a release

**Pull request merged**